src/v0/destinations/google_adwords_offline_conversions/helper.js

```diff
@@ -0,0 +1,6 @@
+const moment = require('moment');
```

Comment:
Generic question:
Can we have this helper in some general util directory or is it very specific to GAOC?

Reply:
Good point 👍🏼 
I think this logic is specific to `GAOC` alone. Not sure if other destinations also require same logic, hence not preferring to move it to a common place for now.